### ISSUE TITLE
Error when typing the " for a git commit message

### ISSUE THREAD

**Issue by Fishezzz** [PowerShell/PSReadLine#1301]
Environment data
----------------
```
PS version: 5.1.18362.145
PSReadline version: 2.0.0-beta2
os: 10.0.18362.356 (WinBuild.160101.0800)
PS file version: 10.0.18362.356 (WinBuild.160101.0800)
HostName: ConsoleHost
BufferWidth: 108
BufferHeight: 3000
```

Steps to reproduce or exception report
--------------------------------------
```
-----------------------------------------------------------------------
Last 59 Keys:
 g i t Space i n i t Enter
 g i t Space a d d Backspace Backspace Backspace s t a t u s Enter
 g i t Space a t Backspace d d Space R Tab Enter
 g i t Space c o m m i t Space - m Backspace S Space - m Space "

Exception:
System.ArgumentOutOfRangeException: De waarde moet groter zijn dan of gelijk zijn aan nul en minder zijn dan de buffergrootte van de console in die dimensie.
Parameternaam: left
Werkelijke waarde was -1.
   bij System.Console.SetCursorPosition(Int32 left, Int32 top)
   bij Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorLeft(Int32 value)
   bij Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   bij Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   bij Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   bij Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   bij Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   bij Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   bij Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
```

This fills up the whole width of the window

```
PS C:\Users\xxxxx\OneDrive - Hogeschool VIVES\3 Derde Jaar\Mobile development\mobile_development_2-commands>
```

so on the next line there are the characters you type.

Maybe this has somethin to do with it, the working directory filling up the whole width of the screen. I don't have any idea tho. It's the first time I have an error that is from powershell itselve.

**Comment by Fishezzz** [comment]
Might be fixed by #967 but I'm still able to recrate the same things here.

**Comment by daxian-dbw** [comment]
This was fixed in recent versions of PSReadLine. Please try out the `2.0.0-rc2` from PowerShell Gallery.
